src/discordbot.js

```diff
@@ -0,0 +1,153 @@
+const { Client, GatewayIntentBits } = require("discord.js");
+const {
+  userSaveNewPosition,
+  deletePosition,
+} = require("./db/savePositionDataDB");
+const { loadAllPositions } = require("./db/loadPositionDataDB");
+const { chains } = require("./utils/chains");
+const logger = require("./utils/logger");
+
+const client = new Client({
+  intents: [
+    GatewayIntentBits.Guilds,
+    GatewayIntentBits.GuildMessages,
+    GatewayIntentBits.DirectMessages,
+  ],
+});
+
+client.on("ready", () => {
+  logger.info("DiscordClient", `Logged in as ${client?.user.tag}!`);
+});
+
+client.on("messageCreate", async (msg) => {
+  // You can view the msg object here with console.log(msg)
+
+  if (
+    msg.content === "" ||
+    (msg.author.bot && msg.author.id !== process.env.DISCORD_TEST_CLIENT_ID)
+  )
+    return;
+
+  const message = String(
+    msg.content.split(`<@${process.env.DISCORD_CLIENT_ID}>`)[1]
+  ).trimStart();
+  const command = message.split(" ")[0];
+  const args = message.split(" ").slice(1);
+  let response;
+  // if (await !isUserExist(msg.author)) {
+  //   msg.reply("User does not exist, plase join first");
+  //   return;
+  // }
+  switch (command) {
+    case "help":
+      msg.reply(
+        "I can help you with the following commands: \n- `GetAllActivePositions` \n- `AddPosition` \n- `RemovePosition`"
+      );
+      break;
+    case "GetAllActivePositions":
+      response = await getAllActivePositions(args);
+      msg.reply(response);
+      break;
+
+    case "AddPosition":
+      response = await addPosition(args);
+      msg.reply(response);
+      break;
+
+    case "RemovePosition":
+      response = await removePosition(args);
+      msg.reply(response);
+      break;
+
+    default:
+      msg.reply(
+        "I don't understand this command, try to use `help` to find posible commands"
+      );
+      break;
+  }
+});
+
+const getAllActivePositions = async (args) => {
```

Comment:
let's also accept ETH and ARB, and also case insensitive

Reply:
what do you mean?

Reply:
Ha like 
 ethereum || ETH || eth ?